app/assets/javascripts/components/dashboard/ownedRecords.jsx

```diff
@@ -19,95 +23,92 @@ class OwnedRecords extends React.Component {
-        </td>
-        <td>
-          <div className="btn-group btn-block" role="group">
-            <button
-              id="btnGroupDrop1"
-              type="button"
-              className="btn btn-primary btn-block dropdown-toggle btn-sm"
-              data-toggle="dropdown"
-              aria-haspopup="true"
-              aria-expanded="false"
-            >
-              <i className="fa fa-cog" />
-            </button>
-            <div className="dropdown-menu" aria-labelledby="btnGroupDrop1">
-              {!this.props.currentUser.canRegisterRecord &&
-                !this.props.currentUser.isAdmin &&
-                <a className="dropdown-item" href={Routes.edit_death_record_path(deathRecord.id)}>
-                  <i className="fa fa-play" />&nbsp;Continue
-                </a>}
-              <a className="dropdown-item" href={Routes.death_record_path(deathRecord.id)}>
-                <i className="fa fa-search" />&nbsp;View
-              </a>
-              {this.props.currentUser.id == deathRecord.creator.id &&
-                <a className="dropdown-item" onClick={() => this.abandonRecord(deathRecord)}>
-                  <i className="fa fa-trash" />&nbsp;Abandon
-                </a>}
-            </div>
-          </div>
-        </td>
-        <td>{deathRecord.id}</td>
-        <td>{this.renderDecedentName(decedentName)}</td>
-        <td>{timeago}</td>
-        <td>{this.renderRecordProgress(deathRecord)}</td>
-      </tr>
-    );
+            <a className="dropdown-item" href={Routes.edit_death_record_path(data.id)}>
+              <i className="fa fa-play" />&nbsp;Continue
+            </a>}
+          <a className="dropdown-item" href={Routes.death_record_path(data.id)}>
+            <i className="fa fa-search" />&nbsp;View
+          </a>
+        </div>
+      </div>)
   }
 
-  abandonRecord(deathRecord) {
-    var self = this;
-    swal(
-      {
-        title: 'You are about to abandon this death record!',
-        text: 'Are you sure?',
-        type: 'error',
-        showCancelButton: true,
-        confirmButtonClass: 'btn-primary',
-        confirmButtonText: 'Abandon!',
-        closeOnConfirm: false,
-        showLoaderOnConfirm: true
-      },
-      function(isConfirm) {
-        if (!isConfirm) return;
-        $.post(Routes.abandon_death_record_path(deathRecord.id));
-      }
-    );
+  renderIdCol(data, type, full, meta) {
+    return data.id
+  }
+
+  renderNameCol(data, type, full, meta) {
+    return ReactDOMServer.renderToStaticMarkup(this.renderDecedentName(this.decedentName(data)))
+  }
+
+  renderTimeagoCol(data, type, full, meta) {
+    return jQuery.timeago(data.lastUpdatedAt)
+  }
+
+  renderProgressCol(data, type, full, meta) {
+    return this.renderRecordProgress(data)
   }
 
   renderRecordProgressIcon(step, deathRecord) {
     if (step.contents.contents && step.contents.requiredSatisfied) {
-      return (
+      return ReactDOMServer.renderToStaticMarkup(
```

Comment:
No, these functions handle rendering for DataTables, so they expect plain HTML back. In this case I am making use of React to build the table cells, but then rendering it to plain HTML so DataTables can use it.